packages/docs/src/routes/docs/(qwikcity)/api/index.mdx

```diff
@@ -276,6 +276,19 @@ The `routeAction$()` function is used to declare a new Server Action in the give
 
 Please refer to the [Server Actions](/docs/(qwikcity)/action/index.mdx) section for more information.
 
+## List all routes with qwikCityPlan.routes
```

Comment:
This title is different from the other in the page but it's more explicate I guess

Reply:
LGTM